src/main/java/com/android/volley/Request.java

```diff
@@ -245,13 +227,14 @@ void finish(final String tag) {
                 // If we finish marking off of the main thread, we need to
                 // actually do it on the main thread to ensure correct ordering.
                 Handler mainThread = new Handler(Looper.getMainLooper());
-                mainThread.post(new Runnable() {
-                    @Override
-                    public void run() {
-                        mEventLog.add(tag, threadId);
-                        mEventLog.finish(Request.this.toString());
-                    }
-                });
+                mainThread.post(
+                        new Runnable() {
```

Comment:
What's the stance on lambdas?

Reply:
I'm hesitant to use them right now just because of how many places Volley code is exported compiled. It *might* work - we'd need recent Gradle plugins which can desugar the Lambdas, and that desugar library would need to work on minSdkVersion 8+. (Or, we'd need to bump Volley's minSdkVersion). For now... I think we should just avoid them.